project_task_material_stock/models/project_task.py

```diff
@@ -90,6 +90,7 @@ def _compute_stock_state(self):
     )
     picking_type_id = fields.Many2one(
         comodel_name="stock.picking.type",
+        domain=[("code", "=", "outgoing")],
```

Comment:
```suggestion
        domain=[("code", "in", ("outgoing", "internal"))],
```

Reply:
If I add internal type, material is not consumed, isn't It?

Reply:
It's consumed from source location and move to destination location, so I think it can be correct.

Reply:
@pedrobaeza done!